sylt-tokenizer/src/tokenizer.rs

```diff
@@ -4,42 +4,99 @@ pub use token::Token;
 
 mod token;
 
-pub type PlacedToken = (Token, usize);
-pub type TokenStream = Vec<PlacedToken>;
-
-pub fn string_to_tokens(content: &str) -> TokenStream {
-    let lexer = Token::lexer(&content);
-
-    let mut placed_tokens = lexer.spanned().peekable();
-
-    let mut lined_tokens = Vec::new();
-    let mut line: usize = 1;
-    for (c_idx, c) in content.chars().enumerate() {
-        if let Some((kind, t_range)) = placed_tokens.peek() {
-            if t_range.start == c_idx {
-                let kind = kind.clone();
-                placed_tokens.next();
-                lined_tokens.push((kind, line));
-            }
-        } else {
-            break;
-        }
+#[derive(Debug, Copy, Clone, PartialEq)]
+/// A location in a file containing source code.
+pub struct Span {
+    // TODO(ed): Do this more intelligent, so
+    // we can show ranges. Maybe even go back
+    // to offsets from start of the file.
+    pub line: usize,
+    /// The first column that this Span contains.
+    pub col_start: usize,
+    /// The first column that this Span doesn't contain.
+    pub col_end: usize,
+}
```

Comment:
Haven't I seen this already? Why not move it to `common`?

Reply:
`sylt-common` depends on `sylt-tokenizer`, and since it's needed in the tokenizer it now lives there.

Reply:
What you saw was the static which has now been moved here as well.